be/src/storage/segment_replicate_executor.cpp

```diff
@@ -161,8 +141,9 @@ void ReplicateChannel::_send_request(SegmentPB* segment, butil::IOBuf& data, boo
         _closure->cntl.request_attachment().append(data);
     }
     _closure->request_size = _closure->cntl.request_attachment().size();
+
     // brpc send buffer is also considered as part of the memory used by load
-    _mem_tracker->consume(_closure->request_size);
+    _mem_tracker->consume_without_root(_closure->request_size);
```

Comment:
This part of the memory is counted in load_memory for the purpose of ingestion back pressure. If the statistics are alone, the effect will not be achieved.

Reply:
The mem_tracker is not one alone mem_tracker, Actually, it is a sub `Memtracker` of `LoadMemTracker`